readme.md

```diff
@@ -9,7 +9,16 @@ If you want to contribute as a developer, please contact us in the Discord. http
 
 ## Building
 
-Under construction.
+1. Install dependencies
+   
+   `sudo apt install build-essential cmake clang libunwind-dev libglfw3-dev`
```

Comment:
Hi! I have tested instructions, they works for me. Thanks! 

btw, for me on fresh Ubuntu 22
```
sudo apt install cmake g++ libunwind-dev libglfw3-dev
```
was enought.

Why clang is needed?

Reply:
Clang is not technically needed, but the linting and formatting is based on clang standard, so it's probably the recommended compiler for RPCSX

Reply:
How much does it weigh on disk?